js/index.js

```diff
@@ -1 +1,101 @@
 // Your code goes here
+const title = document.querySelector('.logo-heading');
+
+title.addEventListener('mouseover',function() {
+    title.style.color = '#FDBA58';    //@bus-yellow
+})
+
+title.addEventListener('mouseout',function() {
+    title.style.color = 'black';
+})
+
+const h2 = document.querySelectorAll('h2');
+
+h2.forEach( function(h2tag) {
+    h2tag.addEventListener('mouseover', () => h2tag.style.color = '#FDBA58' );
+})
+
+h2.forEach( h2tag => h2tag.addEventListener('mouseout', () => h2tag.style.color = 'black'));
+
+const h4 = document.querySelectorAll('h4');
+
+h4.forEach( h4tag => h4tag.addEventListener('mouseover', () => h4tag.style.color = '#FDBA58'));
+
+h4.forEach(h4tag => h4tag.addEventListener('mouseout', () => h4tag.style.color = 'black'));
+
+
+const a = document.querySelectorAll('.nav-link');
+
+a.forEach(atag => atag.addEventListener('mouseover', function() {
+    atag.style.color = '#FDBA58';
+}));
+
+a.forEach(atag => atag.addEventListener('mouseout', function() {
+    atag.style.color = 'black';
+}));
+
+const input = document.querySelector('#input');
```

Comment:
When we are selecting an ID, it's usually best to do `getElementById` instead of querySelector, as getElementById is much more optimized.